account_invoice_line_sequence/__manifest__.py

```diff
@@ -5,15 +5,19 @@
 
 {
     "name": "Account Invoice line with sequence number",
-    "version": "14.0.1.0.1",
+    "version": "14.0.2.0.1",
```

Comment:
The migration script should not be in a directory including the last digit ?
"14.0.2.0.1" vs "14.0.2.0"

Reply:
indeed, corrected